src/seedu/addressbook/data/AddressBook.java

```diff
@@ -114,6 +114,11 @@ public UniquePersonList getAllPersons() {
         return new UniquePersonList(allPersons);
     }
 
+    /**
+     * Sorts all persons by alphabetical order of their name
+     */
+    public void sortAllPersons(){ allPersons.sort(); }
```

Comment:
1. Please take a look at the coding standards and see how to write methods with a single line of code.
2. Need space between `) {`